docs/en/operations/system-tables/data_skipping_indices.md

```diff
@@ -8,12 +8,11 @@ Columns:
 -   `table` ([String](../../sql-reference/data-types/string.md)) — Table name.
 -   `name` ([String](../../sql-reference/data-types/string.md)) — Index name.
 -   `type` ([String](../../sql-reference/data-types/string.md)) — Index type.
--   `expr` ([String](../../sql-reference/data-types/string.md)) — Expression used to calculate the index.
--   `granularity` ([UInt64](../../sql-reference/data-types/int-uint.md)) — Number of granules in the block.
+-   `expr` ([String](../../sql-reference/data-types/string.md)) — The expression is used to calculate the index.
```

Comment:
I'd prefer to avoid passive voice here.
```suggestion
-   `expr` ([String](../../sql-reference/data-types/string.md)) — Expression for the index calculation.
```